libs/zzz/svgicons/src/icons/Slots/SlotFiveIcon.tsx

```diff
@@ -0,0 +1,31 @@
+import { createSvgIcon } from '@mui/material'
+
+export const SlotFiveIcon = createSvgIcon(
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    width="24px"
+    height="24px"
+    viewBox="0 0 24 24"
+    version="1.1"
+  >
+    <defs>
+      <symbol
+        xmlns="http://www.w3.org/2000/svg"
+        fill="none"
+        viewBox="0 0 24 24"
+        id="gti--zzz-game-mindscape-0142391894no66qoo3m3d"
+      ></symbol>
+    </defs>
+    <g id="surface1">
+      <path
+        style={{
+          stroke: 'none',
+          fillRule: 'nonzero',
+          fill: '#FF9100',
```

Comment:
i think you can remove all but the `<path/>` here, and you can likely remove the `path.style` here as well (the `fill` may prevent this svg from inheriting color for non S-rank slots)
```
const Icon = createSvgIcon(
  <path d="..." />,
  'Name',
);
```